src/Core/src/Platform/Windows/MauiRadioButton.cs

```diff
@@ -0,0 +1,78 @@
+﻿#nullable disable
+using Microsoft.UI.Xaml;
+using WBrush = Microsoft.UI.Xaml.Media.Brush;
+using WContentPresenter = Microsoft.UI.Xaml.Controls.ContentPresenter;
+
+namespace Microsoft.Maui.Platform
+{
+	public class MauiRadioButton : UI.Xaml.Controls.RadioButton
+	{
+		public static readonly DependencyProperty BorderRadiusProperty = DependencyProperty.Register(nameof(BorderRadius), typeof(int), typeof(MauiRadioButton),
+			new PropertyMetadata(default(int), OnBorderRadiusChanged));
+
+		public static readonly DependencyProperty BackgroundColorProperty = DependencyProperty.Register(nameof(BackgroundColor), typeof(WBrush), typeof(MauiRadioButton),
+			new PropertyMetadata(default(WBrush), OnBackgroundColorChanged));
+
+		WContentPresenter _contentPresenter;
+
+		public WBrush BackgroundColor
+		{
+			get
+			{
+				return (WBrush)GetValue(BackgroundColorProperty);
+			}
+			set
+			{
+				SetValue(BackgroundColorProperty, value);
+			}
+		}
+
+		public int BorderRadius
+		{
+			get
+			{
+				return (int)GetValue(BorderRadiusProperty);
+			}
+			set
+			{
+				SetValue(BorderRadiusProperty, value);
+			}
+		}
+
+		protected override void OnApplyTemplate()
+		{
+			base.OnApplyTemplate();
+
+			_contentPresenter = GetTemplateChild("ContentPresenter") as WContentPresenter;
+
+			UpdateBackgroundColor();
+			UpdateBorderRadius();
+		}
+
+		static void OnBackgroundColorChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+		{
+			((MauiRadioButton)d).UpdateBackgroundColor();
+		}
+
+		static void OnBorderRadiusChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+		{
+			((MauiRadioButton)d).UpdateBorderRadius();
+		}
+
+		void UpdateBackgroundColor()
+		{
+			if (BackgroundColor == null)
+				BackgroundColor = Background;
+
+			if (_contentPresenter != null)
+				_contentPresenter.Background = BackgroundColor;
```

Comment:
Seems we need to keep track of the focused visual state, as when you hover over the radiobutton, the border disappears.

Reply:
Applied changes to fix the wrong behavior with the mouse hover state.